code-challenges.js

```diff
@@ -31,24 +36,57 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const boilingPoint = 212
+const checkWaterTemp = (temp) => {
+    if (temp < boilingPoint) {
+        return `${temp} is below boiling point`
+    } else if (temp > boilingPoint){
+        return `${temp} is above boiling point`
+    } else if (temp === boilingPoint) {
+        return `${temp} is at boiling point`
+    } else {
+        return "Something is not right"
+    }
+}
```

Comment:
Awesome job on this one! Love that you accounted for something going wrong and used your catch all. 💯 You can also mention string interpolation in your pseudocode to help be as descriptive as possible.